.github/workflows/go.yml

```diff
@@ -7,17 +7,22 @@ on:
     branches: [ master ]
 
 jobs:
-
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
 
     - name: Set up Go
-      uses: actions/setup-go@v2
+      uses: actions/setup-go@v5
       with:
         go-version: 1.17
```

Comment:
Missing the bump from this https://github.com/aayushduwadi/aeron-go/pull/3/files#diff-678682767f2477de3e3c546746f8568b9a1942b2c647d32331d7e774b8ff8d9fR18

Reply:
looks like the bump was done here:
https://github.com/talostrading/aeron-go/pull/51/commits/b027c598497a3d5823868b274138f5d12b406326

should I only pull golang bump here?